server/src/pages/shot/view.js

```diff
@@ -212,7 +212,7 @@ class Body extends React.Component {
         <div>
           <div className="spacer"/>
           If you do nothing,<br/>
-          this shot will be permanently deleted <TimeDiff date={deleteTime} />.
+          this shot will be permanently deleted in <TimeDiff date={deleteTime} />.
```

Comment:
Note the TimeDiff stuff is all going to have to be changed quite a bit for localization